pkg/services/live/pipeline/config.go

```diff
@@ -61,12 +63,14 @@ type FrameOutputterConfig struct {
 	ConditionalOutputConfig *ConditionalOutputConfig   `json:"conditional,omitempty"`
 	ThresholdOutputConfig   *ThresholdOutputConfig     `json:"threshold,omitempty"`
 	RemoteWriteOutputConfig *RemoteWriteOutputConfig   `json:"remoteWrite,omitempty"`
+	LokiOutputConfig        *LokiOutputConfig          `json:"loki,omitempty"`
```

Comment:
I see `LokiOutputConfig` as both a `Frame-` and a `DataOutputter`, is it about sending the data to Loki in different formats?